.example.env

```diff
@@ -4,16 +4,14 @@ CDK_DEFAULT_REGION=[REQUIRED IF DEPLOYING TO EXISTING VPC]
 
 STAGE=[FILL ME IN]
 
-VEDA_DB_PGSTAC_VERSION=0.7.10
+VEDA_DB_PGSTAC_VERSION=0.9.1
```

Comment:
FYI: stac-fastapi-pgstac is not yet compatible with pgstac 0.9.1 

I would suggest to first move to 0.8.5